elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -76,35 +97,56 @@ defmodule API.Client.Channel do
   end
 
   def handle_info({:resource_removed, resource_id}, socket) do
+    OpenTelemetry.Tracer.set_current_span(socket.assigns.opentelemetry_ctx)
+    OpenTelemetry.Tracer.add_event("resource_updated", %{resource_id: resource_id})
+
     push(socket, "resource_removed", resource_id)
     {:noreply, socket}
   end
 
+  def handle_in("create_log_sink", _attrs, socket) do
```

Comment:
@thomaseizinger this is the message that you need to send from a client to get a signed URL where logs can be uploaded.